internal/characters/keqing/skill.go

```diff
@@ -97,7 +100,7 @@ func (c *char) skillRecast(p map[string]int) action.ActionInfo {
 		}
 		// TODO: this should be 1st hit on cast and 2nd at end
 		for i := 0; i < hits; i++ {
-			c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 2), skillRecastHitmark, skillRecastHitmark)
+			c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), skillC1Radius[i]), skillRecastHitmark, skillRecastHitmark)
```

Comment:
c1 radius is 2 at start and 2 at terminus, no need for a slice
1st hit should also center on PrimaryTarget if you let E-LightningStello center on PrimaryTarget